luarules/gadgets/game_team_power_watcher.lua

```diff
@@ -41,15 +41,12 @@ local powerThresholds = {
     {techLevel = 4.5, threshold = 725000}
 }
 
+
+local scavengerTeam = Spring.Utilities.GetScavTeamID()
+local raptorTeam = Spring.Utilities.GetChickenTeamID()
 for _, teamID in ipairs(teamList) do
-    local teamLuaAI = Spring.GetTeamLuaAI(teamID)
     local allyID = select(6, Spring.GetTeamInfo(teamID))
-
-    if (teamLuaAI and string.find(teamLuaAI, "ScavengersAI")) then
-        scavengerTeam = teamID
-    elseif (teamLuaAI and string.find(teamLuaAI, "RaptorsAI")) then
-        raptorTeam = teamID
-    elseif select (4, Spring.GetTeamInfo(teamID, false)) then
+    if select (4, Spring.GetTeamInfo(teamID, false)) then
```

Comment:
This still needs to exclude the Raptors and Scavengers from this if statement, otherwise they will get counted among the regular AI teams.

Reply:
does the new code solve it?